client/app/components/composites/MenuMobile/LanguagesMobile.js

```diff
@@ -0,0 +1,55 @@
+import { Component, PropTypes } from 'react';
+import r, { div, span } from 'r-dom';
+import Link from '../../elements/Link/Link';
+import css from './MenuMobile.css';
+import globeIcon from './images/globeIcon.svg';
+import variables from '../../../assets/styles/variables';
+
+class LanguagesMobile extends Component {
+
+  render() {
+    const languages = this.props.links ?
+      this.props.links.map((link) => {
+        const linkColor = link.active ?
+          variables['--LanguagesMobile_textColorSelected'] :
+          this.props.color || variables['--LanguagesMobile_textColorDefault'];
+        return div(
+          { className: `LanguagesMobile_languageLink ${css.languageLink}` },
+          r(Link, { href: link.href, customColor: linkColor }, link.content));
+      }) :
+      [];
+
+    return languages.length > 0 ?
```

Comment:
@Gnito actually the check for multiple languages is done in Topbar.js https://github.com/sharetribe/sharetribe/pull/2260/files#diff-dadeb88347d45d99fce790d221cef12aR174
which probably makes this check redundant though.